mover-restic/entry.sh

```diff
@@ -26,6 +26,15 @@ function check_var_defined {
     fi
 }
 
+function check_contents {
+echo "== Checking directory for content ==="
+# shellcheck disable=SC2086
+if [ -z "$(ls -A ${DATA_DIR})" ]; then
```

Comment:
Perhaps there's something strange going on w/ putting it all in the test?
```bash
$ EDIR=/home/jstrunk/src/backube/scribe/empty
$ R="$(ls -A "${EDIR}")"
$ $([ -z "$R" ])
$ echo $?
0
$ EDIR=/home/jstrunk/src/backube/scribe
$ R="$(ls -A "${EDIR}")"
$ $([ -z "$R" ])
$ echo $?
1
```
While it doesn't need to be quoted based on the current usage (it's always `/data`), it really should be in case we end up w/ a space in there at some point.

Also, please fix the indenting of the function.

Reply:
Sorry, I don't think I was clear. I didn't mean to hardcode `/data`. That path is determined by the operator, so it may change in the future. I was trying to suggest breaking the line up into something like:
```bash
DIR_CONTENTS="$(ls -A "${DATA_DIR}")"
if [ -z "${DIR_CONTENTS}" ]; then
```
...since it would provide the proper quoting of the path and avoid the need to have the shellcheck ignore. (Not that I understand why the original quoted form, putting it all in the test wouldn't work)